addons/event/models/res_config_settings.py

```diff
@@ -1,12 +1,23 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, fields, models
+import base64
+import binascii
 
+from odoo import _, api, exceptions, fields, models
 
 class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
+    def _default_use_google_maps_static_api(self):
+        api_key = self.env['ir.config_parameter'].sudo().get_param('mail.google_maps_static_api_key')
```

Comment:
Not sure about the name myself. It's not technically from mail but it seems more relevant to it.
```suggestion
        api_key = self.env['ir.config_parameter'].sudo().get_param('event.google_maps_static_api_key')
```

Reply:
I would just call it `google_maps.static_api_key` (even if the module doesn't exists, but maybe one day ?)